pkg/operator/ceph/object/user/controller.go

```diff
@@ -252,7 +252,13 @@ func (r *ReconcileObjectStoreUser) reconcile(request reconcile.Request) (reconci
 	}
```

Comment:
Commitment complains, Change the commit to 

object: add ssl ref in cephobjectstore user secret